core/spec/models/mno_enterprise/user_spec.rb

```diff
@@ -385,6 +413,68 @@ def reload_user
       end
     end
 
+    describe '.requires_otp_for_login?' do
+      let(:user) { build(:user) }
+
+      describe '#admin?' do
```

Comment:
- Prefer context to describe the context
- Avoid stubbing too much (we have an [admin trait on the factory](https://github.com/maestrano/mno-enterprise/blob/5.0/core/lib/mno_enterprise/testing_support/factories/users.rb#L46-L48))

This can be rewritten this way

```ruby
describe '.requires_otp_for_login?' do
  subject { user.requires_otp_for_login? }
  
  context "when the user is an admin" do
    let(:user) { build(:user, :admin) }

    context 'when 2FA is required for admin' do
      before { Settings.authentication.two_factor.admin_enabled = true }

      it { is_expected.to be true }
    end

    context 'when 2FA is not required for admin' do
      before { Settings.authentication.two_factor.admin_enabled = false }

      it { is_expected.to be false }
    end
  end
end
```